### PR TITLE
Added Cursor pointer on menu and link hover

### PR DESCRIPTION
**Pull request by abanseka**
cursor changes to pointer when 
- hamburger menu button is hovered
- nav links are hovered

### CODE REVIEW

File: src/Components/Header/Header.css [hunk 1]
```diff
@@ -99,7 +104,7 @@
     position: absolute;
     z-index: 2;
     width: 100%;
-    height: 30vh;
+    height: 50vh;
```
Comment (abanseka) [inline src/Components/Header/Header.css:107]: The nav links overlapped the hamburger-menu-button
making it difficult to be clicked. 
> reason for added height.

**Comment by ahamad-ch** [comment]
Next time when you're starting to work on something new, make sure you create a new branch and then add a PR.

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -8,35 +9,82 @@ import In from "../../images/In.png";
...
+            </p>
+            <p className="text-description footer-link-text">
+              <scroll.Link to="partners" smooth={true}>
+                About
+              </scroll.Link>
+            </p>
+            <p className="text-description footer-link-text">
+              <scroll.Link to="header" smooth={true}>
+                Contact
+              </scroll.Link>
```
Comment (ahamad-ch) [inline src/Components/Footer/Footer.js:37]: You can import `{ Link} from "react-scroll"` and then use `<Link to="gallery" smooth={true}>`

**Review by ahamad-ch — CHANGES_REQUESTED** [review]

File: src/Components/Footer/Footer.js [hunk 3]
```diff
@@ -8,35 +9,82 @@ import In from "../../images/In.png";
...
+            <p className="text-description footer-link-text">
+              <Link to="partners" smooth={true}>
+                About
+              </Link>
+            </p>
+            <p className="text-description footer-link-text">
+              <Link to="header" smooth={true}>
+                Contact
+              </Link>
+            </p>
```
Comment (ahamad-ch) [inline src/Components/Footer/Footer.js:38]: As the links looks a bit expanded now, lets add the links inside a loop.
You can use a variable with the values needed.
Example:
```
const links = [{
  label: "Home",
  to: "features"
}, {
  label: "About",
  to: "partners"
}]
```

**Review by ahamad-ch — APPROVED** [review]

**Pull request merged**